packages/kit/src/core/create_manifest_data/index.js

```diff
@@ -50,9 +50,10 @@ export default function create_manifest_data({ config, output, cwd = process.cwd
 	 * @param {string} dir
 	 * @param {Part[][]} parent_segments
 	 * @param {string[]} parent_params
-	 * @param {string[]} stack
+	 * @param {string[]} layout_stack
```

Comment:
it'd be helpful to document what these are since it's not obvious from the type

Reply:
done